blog/_posts/2019-04-24-MLJ.md

```diff
@@ -0,0 +1,253 @@
+<!---
+![](https://github.com/JuliaLang/www.julialang.org/tree/master/images/blog/2019-04-26-MLJ/learningcurves.png)
```

Comment:
?

Reply:
not sure to be honest, this was anthony's way of making the positioning correct I think